### ISSUE TITLE
fix_weight_class variable font improvement

### ISSUE THREAD

**Issue by moontypespace** [googlefonts/gftools#477]
According to Microsoft's OT spec the usWeightClass should match the fvar default value. 
In most of the cases this might be 400, but in some it's not.

fix_weight_class does not take care of that:
https://github.com/googlefonts/gftools/blob/f0d02b32af0ee57c5df2b9ca683b071a9c029a44/Lib/gftools/fix.py#L184

The OT Spec says:
_In a variable font that implements 'wght' variations, the value in the usWeightClass field of the OS/2 table must match the default 'wght' value specified in the 'fvar' table._
https://docs.microsoft.com/en-us/typography/opentype/spec/dvaraxistag_wght#additional-information

That do you think about something like that:
```
def fix_weight_class_var(font_obj):
    if 'fvar' not in font_obj:
        # not a variable font
        return

    fvar = font_obj['fvar']
    default_axis_values = {a.axisTag: a.defaultValue for a in fvar.axes}

    print('default_axis_values: ', default_axis_values)
    v = default_axis_values.get('wght', None)

    if isinstance(v, int) or isinstance(v, float):
        font_obj["OS/2"].usWeightClass = int(v)
        print('Changed usWeightClass tom match fvar default value.')
```

**Comment by simoncozens** [comment]
This sounds like a good idea. We should also have a fontbakery check which does this.

By the way, the second `if` test could just be `if v is not None`.

**Comment by moontypespace** [comment]
@simoncozens I just made a PR :) I will also add that to fontbakery.

**Comment by m4rc1e** [comment]
Fixed in #481
